Health/src/main/java/com/sciencesquad/health/activity/MapsActivity.java

```diff
@@ -314,4 +323,43 @@ public void onLocationChanged(Location location) {
         handleNewLocation(location);
 
     }
+
+    /**
+     * I have not been tested :-)
+     */
+    @SuppressWarnings("StatementWithEmptyBody")
+    @Override
+    public boolean onNavigationItemSelected(MenuItem item) {
```

Comment:
Please don't do this. :cry: Remove all of these from everything except `MainActivity`.